modules/RTC/TPCUtils.js

```diff
@@ -363,20 +368,17 @@ export class TPCUtils {
         // Find the first recvonly transceiver when more than one track of the same media type is being added to the pc.
         // As part of the track addition, a new m-line was added to the remote description with direction set to
         // recvonly.
-        } else if (FeatureFlags.isMultiStreamSupportEnabled()
-            && this.pc.getLocalTracks(mediaType)?.length
-            && !newTrack.conference) {
+        } else if (isNewSecondaryLocalSource) {
             transceiver = this.pc.peerconnection.getTransceivers().find(
                 t => t.receiver.track.kind === mediaType
                 && t.direction === MediaDirection.RECVONLY
                 && t.currentDirection === MediaDirection.INACTIVE);
 
-        // For unmute operations, find the transceiver based on the track index in the source name if present, otherwise
-        // it is assumed to be the first local track that was added to the peerconnection.
+        // For mute/unmute operations, find the transceiver based on the track index in the source name if present,
+        // otherwise it is assumed to be the first local track that was added to the peerconnection.
```

Comment:
is it possible to always use the index instead of having a special case for "the first local track that was added to the peerconnection"?

Reply:
This is needed for the clients that don't support source name signaling and send only one track per media type.